src/cclib/method/opa.py

```diff
@@ -21,11 +21,19 @@ def func(x):
         return x+func(x-1)
 
 
-class OPA(Method):
+class OPA(Population):

```

Comment:
@ATenderholt This should be a subclass of `Population` and not of `Method` right?

Reply:
Done in #642.